src/internal/m365/exchange/restore_test.go

```diff
@@ -369,3 +373,71 @@ func (suite *RestoreIntgSuite) TestRestoreExchangeObject() {
 		})
 	}
 }
+
+func (suite *RestoreIntgSuite) TestRestoreAndBackupEventWithRecurringAndAttachments() {
```

Comment:
Nit: please follow golang example naming conventions for test names.  ie: `TestPrimaryFunc_variationOfTest`
```suggestion
func (suite *RestoreIntgSuite) TestRestoreAndBackupEvent_recurringInstancesWithAttachments() {
```